windows/src/Defines.mak

```diff
@@ -1,5 +1,7 @@
 # DEBUG=1
 
+KEYMAN_VERSION=11.0
```

Comment:
So we are not drawing this from VERSION.md?

Reply:
Looks like there are a few other places in the Windows code also ignoring it.

Reply:
We can't at present. We'd have to build a file to read it. Not worth the effort.

Reply:
Also, the Windows files need deliberation w.r.t. the keyboard versioning as well as product versioning. Better to document these in the version upgrade docs I think than to automate.